graphql/queryResolvers/moduleCrud.ts

```diff
@@ -0,0 +1,29 @@
+import type { MutationDeleteModuleArgs, MutationAddModuleArgs } from '..'
+import prisma from '../../prisma'
+
+export const modules = async () => {
+  return prisma.module.findMany({
+    include: {
+      author: true
+    }
+  })
+}
+
+export const addModule = async (_parent: void, args: MutationAddModuleArgs) => {
+  const { authorId, content, lessonId, name } = args
+  if (!authorId || !content || !name || !lessonId) {
+    throw new Error('Missing parameters')
+  }
+  return await prisma.module.create({
```

Comment:
What if this results in error? We should wrap this in a try/catch block

Reply:
if this returns an error the front end will get an error, which could render a message. What do you suggest we do in the `catch` @anthonykhoa ?

Reply:
Ah i see that makes sense. In this case I think we should only log the error then.
Something that tells us creation of module failed, for example `MODULE_CREATION_FAILED_ARGS:${JSON.stringify(args)}`

Reply:
Hmm wait hold on

Reply:
If we don't want to return an error to the frontend, I guess we can return some default value like `{}`.

But I think we should throw the error. And then we handle the error from the frontend too. If there is an error thrown in this resolver, we can have error notifications popup on the screen saying something like `Module Creation Failed`.

Reply:
Apollo already provides `onError` options with their useMutation and useQuery hooks. 
https://www.apollographql.com/docs/react/data/error-handling/